players/factories.py

```diff
@@ -1,6 +1,10 @@
 import factory
 
+from accounts.factories import UserFactory
+
 
 class PlayerFactory(factory.django.DjangoModelFactory):
+    user = factory.SubFactory(UserFactory)
```

Comment:
Because of this change, basically all tests still pass. Hooray for using factories.